docs/setup/turn/eturnal.md

```diff
@@ -0,0 +1,153 @@
+can be used instead. There is an official [AlpineLinux package](https://pkgs.alpinelinux.org/packages?name=eturnal) 
+available. On other Linux systems, the binary release can be installed as [described](https://eturnal.net/documentation/#Installation) 
+in the reference documentation. For Windows, an installer is [available](https://eturnal.net/windows/). 
+On other platforms, `eturnal` is [built from source](https://github.com/processone/eturnal/blob/master/INSTALL.md).
+
+### Configuration
+
+1.  Create or edit the config file in `/etc/eturnal.yml`. 
+    See the [example configuration](https://github.com/processone/eturnal/blob/master/config/eturnal.yml) 
+    for explanations of the options.
+    
+    The relevant lines, with example values, are:
+
+    ```yaml
+    eturnal:
+        secret: "long-and-cryptic"     # Shared secret, CHANGE THIS.
+    ```
+
+    One way to generate the `secret` is with `pwgen`:
+
+    ```sh
+    pwgen -s 64 1
+    ```
+
+1.  If your TURN server is behind NAT, the NAT gateway must have an external,
+    publicly-reachable IP address. `eturnal` tries to autodetect the public IP address, however, it may also be configured, so `eturnal` advertises that
+    address to connecting clients:
+
+    ```yaml
+        relay_ipv4_addr: "203.0.113.4" # The server's public IPv4 address.
+    ```
+
+    If your NAT gateway is reachable over both IPv4 and IPv6, you may
+    configure `eturnal` to advertise each available address:
+
+    ```yaml
+        relay_ipv4_addr: "203.0.113.4" # The server's public IPv4 address.
+        relay_ipv6_addr: "2001:db8::4" # The server's public IPv6 address (optional).
+    ```
+
+    When advertising an external IPv6 address, ensure that the firewall and
+    network settings of the system running your TURN server are configured to
+    accept IPv6 traffic, and that the TURN server is listening on the local
+    IPv6 address that is mapped by NAT to the external IPv6 address.
+
+1.  If eturnal was started by systemd, log files are written into the
+    `/var/log/eturnal` directory by default. In order to log to the [journal](https://www.freedesktop.org/software/systemd/man/systemd-journald.service.html)
+    instead, the `log_dir` option can be set to `stdout` in the configuration file.
+
+1.  Consider your security settings. TURN lets users request a relay which will
+    connect to arbitrary IP addresses and ports. The following configuration is
+    suggested as a minimum starting point, [see also the official documentation](https://eturnal.net/documentation/#blacklist):
+
+    ```yaml
+        ## Reject TURN relaying from/to the following addresses/networks:
+        blacklist:                # This is the default blacklist.
+            - "127.0.0.0/8"         # IPv4 loopback.
+            - "::1"                 # IPv6 loopback.
+            #- recommended          # Expands to a number of networks recommended to be
```

Comment:
we should uncomment this by default, try to match the coturn example where we include private-use IP addresses.
```suggestion
            - recommended          # Expands to a number of networks recommended to be
```

It's good that you make a note about it below, but I'd favour 'copy & paste safety' since a lot of people are likely to not pay too much attention to what they're copying whilst trying to get it working. Then when it works, they'll forget about things like this. I know that's not *our* fault but we can at least try our best to not drop them in trouble.

Reply:
(if you can point people to the right way to whitelist a network, even if it's just by linking to the eturnal docs, that would be really kind)